cmd/thanos/tools_bucket.go

```diff
@@ -619,7 +642,56 @@ func registerBucketCleanup(app extkingpin.AppClause, objStoreConfig *extflag.Pat
 	})
 }
 
-func printTable(blockMetas []*metadata.Meta, selectorLabels labels.Labels, sortBy []string) error {
+type tablePrinter func(w io.Writer, t Table) error
+
+func printTable(w io.Writer, t Table) error {
+	table := tablewriter.NewWriter(w)
+	table.SetHeader(t.Header)
+	table.SetBorders(tablewriter.Border{Left: true, Top: false, Right: true, Bottom: false})
+	table.SetCenterSeparator("|")
+	table.SetAutoWrapText(false)
+	table.SetReflowDuringAutoWrap(false)
+	table.SetAlignment(tablewriter.ALIGN_LEFT)
+	table.AppendBulk(t.Lines)
+	table.Render()
+	return nil
+}
+
+func printCSV(w io.Writer, t Table) error {
+	csv := csv.NewWriter(w)
+	err := csv.Write(t.Header)
+	if err != nil {
+		return err
+	}
+	err = csv.WriteAll(t.Lines)
+	if err != nil {
+		return err
+	}
```

Comment:
Tiny nit - maybe this is a good place to scope the errors to the if checks?
```suggestion
	if err := csv.Write(t.Header); err != nil {
		return err
	}
	if err := csv.WriteAll(t.Lines); err != nil {
		return err
	}
```